src/Common/Config/Kremlin.cpp

```diff
@@ -152,6 +163,19 @@ bool Kremlin::CreateGenericCommandGroups(bool lastTry) {
 			if (missingCommand) {
 				continue;
 			}
+			YAML::Node defaultConfig;
+			if (defaultConfigs.count(fullName) == 0) {
+				if (lastTry) {
+					defaultConfig = YAML::Node();
+				} else {
+					continue;
+				}
+			} else {
+				defaultConfig = defaultConfigs[fullName];
+			}
+			for (YAML::Node subnode : defaultConfig) {
+				commandConfig.push_back(subnode);
+			}
```

Comment:
Also, I believe that the way this is implemented, it only works for sequences and crashes if the command config isn't a sequence, and also the default config would override the command config which isn't what we want.